src/structures/interfaces/InteractionResponses.js

```diff
@@ -162,6 +162,7 @@ class InteractionResponses {
    * @returns {Promise<Message|APIMessage>}
    */
   followUp(options) {
```

Comment:
Either make this function async, or wrap the Error into a `return Promise.reject`